corporate/views/installation_activity.py

```diff
@@ -157,6 +158,13 @@ def realm_summary_table() -> str:
                     AND subgroup = 'true'
                     AND end_time = %(active_users_audit_end_time)s
             ) as bot_count_table ON realm.id = bot_count_table.realm_id
+            LEFT OUTER JOIN (
+                SELECT
+                    extra_data->>'how_did_you_find_zulip' as how_did_you_find_zulip,
+                    realm_id
+                from
+                    zerver_realmauditlog
+            ) as realm_audit_log_table ON realm.id = realm_audit_log_table.realm_id
```

Comment:
This join looks like it's missing a `WHERE` clause limiting to just the realm-creation event type?

Reply:
Yeah, added.